tests/test_perf_analyzer_config_generator.py

```diff
@@ -347,6 +347,129 @@ def test_min_concurrency(self):
         self._run_and_test_perf_analyzer_config_generator(
             yaml_str, expected_configs, pa_cli_args)
 
+    def test_request_rate_range_list(self):
+        """
+        Test Request Rate Range:
+            - Shmoo RRR
+            - Test with auto-search enabled & disabled
+
+        RRR: 1,2,3,4
+        Default (1) value will be used for batch size
+        and 4 configs will be generated
+        """
+
+        # yapf: disable
+        yaml_str = ("""
+            profile_models:
+                - my-model
+            """)
+        # yapf: enable
+
+        request_rate_ranges = [1, 2, 3, 4]
+        expected_configs = [
+            construct_perf_analyzer_config(request_rate_range=rrr)
+            for rrr in request_rate_ranges
+        ]
+
+        pa_cli_args = ['-rrr', '1,2,3,4']
```

Comment:
is `-rrr` an PA thing, and MA thing, or just something made up for testing?

Reply:
Removed `-rrr`